auth/client-api.md

```diff
@@ -18,9 +18,16 @@ abstract class Client {
 
   // query all pending requests
   public abstract getPendingRequests(): Promise<Record<number, PendingRequest>>;
+  
+  // querry pairings
+  public abstract getPairings(): Promise<Array<Pairing>>;
+  
+  // for either to ping a peer in a pairing
+  public abstract ping(params: { topic: string; }): Promise<void>;
+  
+  // for either to disconnect a pairing
+  public abstract disconnect(params: { topic: string, reason: Reason}): Promise<void>;
```

Comment:
Should we add `Reason` to the data structures?

Reply:
actually, the reason will be always .userDisconnected 
so what about:
```suggestion
  public abstract disconnect(params: { topic: string }): Promise<void>;
```

Reply:
@bkrem ⬆️

Reply:
True since it’s an explicit top level action here it’s always user initiated 👍